<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="cur_dev_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current device:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cur_dev_lineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="cur_actual_dev_label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Currently active devices:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="config_button">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>70</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Configure device</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restart_dev_button">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>310</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Restart device</string>
    </property>
   </widget>
   <widget class="QPushButton" name="set_calib_button">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>190</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Set calibration data</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_button">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>430</y>
      <width>271</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>OTA update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggle_all_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>550</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Toggle all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="all_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="device_table">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>681</width>
      <height>361</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>true</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>169</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>159</number>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Selection state</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Location</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MAC</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Firmware</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
